TedTalks/TedTalks/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We might want to use `.aspectFit` to fill the whole thumbnail `UIImageView` so UI looks consistent even for images with different aspect ratios

Reply:
It was already with Aspec fit, instead I used Scale to fill

Reply:
Sorry I meant `.aspectFill` :)